SchoolDay.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <iostream>
+#include "LeakedObjectDetector.h"
+
+struct SchoolDay
+{   
+    SchoolDay();
+    ~SchoolDay();
+    std::string dayOfTheWeek;
+    float length;
+    int numberOfKids = 15;
+    bool runningLate;
+    bool halfDay = false;
+    bool isRaining = false;
+
+    void dayLengthCheck()
+    {
+        std::cout << "the day is now: " <<this->length <<" hours long" << std::endl;
+    }
+    void howManyKids()
+    {
+        std::cout << this->numberOfKids <<" kids in each group " << std::endl;
+    }
```

Comment:
No in-class implementations!
move this to the cpp file
add space around `<<` to make your code more readable
- [ ] revise